runner/cf-driver-go/cmd/drive/drive.go

```diff
@@ -0,0 +1,50 @@
+package drive
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	cobra.EnableCommandSorting = false
+	DriveCmd.AddCommand(configCmd, prepareCmd, runCmd, cleanupCmd)
+}
+
+var DriveCmd = &cobra.Command{
+	Use:   "drive",
```

Comment:
[question] is there a planned use for calling `drive` by itself ever? Feels like we could attach its subcommands directly to root.go instead

Reply:
My thinking was to segregate the `gitlab-runner` stage commands so that there was room in the future to add other stuff to the executable. I'm not sure that would necessarily happen—I was imagining some kind of administrative / configurative command could come in, a `doctor` or who knows. It could be that that never happens, but given that nobody is actually going to type these often, it seemed pretty low-cost?

I'm not super attached to the idea though.

Reply:
That makes sense, and yeah, since these are basically never typed I'm not opposed to the extra characters.